nimbus-features/creditCardAutofillFeature.yaml

```diff
@@ -9,7 +9,7 @@ features:
     defaults:
       - channel: developer
         value: {
-            "credit-card-autofill-status": true
```

Comment:
Did you mean to push this change up?

Reply:
Good catch, I was testing locally and pushed it 😓